github/orgs.go

```diff
@@ -65,6 +65,9 @@ type Organization struct {
 	MembersCanCreatePrivateRepos  *bool `json:"members_can_create_private_repositories,omitempty"`
 	MembersCanCreateInternalRepos *bool `json:"members_can_create_internal_repositories,omitempty"`
 
+	// MembersCanForkPrivateRepositories  toggles whether organization members can fork private organization repositories.
+	MembersCanForkPrivateRepositories *bool `json:"members_can_fork_private_repositories,omitempty"`
```

Comment:
```suggestion
	MembersCanForkPrivateRepos *bool `json:"members_can_fork_private_repositories,omitempty"`
```

Reply:
Thank you so much! I've shortend the name and run `go generate/test/vet` at https://github.com/google/go-github/pull/2389/commits/b510ad5fd6341045d97c761c38b07c337905d0d4.